testnet/stacks-node/src/tests/nakamoto_integrations.rs

```diff
@@ -6424,15 +6424,17 @@ fn check_block_info_rewards() {
         vec![&clarity::vm::Value::UInt(last_nakamoto_block)],
     );
     let tuple0 = result0.expect_tuple().unwrap().data_map;
-    assert_eq!(
+    assert!(
         tuple0
             .get("block-reward")
             .unwrap()
             .clone()
             .expect_optional()
             .unwrap()
-            .unwrap(),
-        Value::UInt(3061200000)
+            .unwrap()
```

Comment:
Isn't this a tautology?  How do you know that the block reward value is correct?

Reply:
I don't think I can be sure exactly what it will be, because of missed tenures that show up, so I changed it to just check for > 0. Is there a way to know exactly what it should be?

Reply:
I think the way around this is to calculate based on the number of tenures. Basically, in the test code, query for the number of tenures, and then compute what the expected value should be, and then use that in the assertion against the clarity value.

Reply:
Added in 1cca58e9b.